PythonAPI/examples/autoware_demo.py

```diff
@@ -288,6 +302,109 @@ def move_spectator(world, ego_vehicle):
 
     spectator.set_transform(spectator_tf)
 
+
+def apply_world_settings(client, world, map_name=None):
+    """
+    Stores all settings related to the simulation world.
+    Applies Synchronous mode + fixed time-step into world settings.
+
+    :param client: Connected client to the Carla server instance.
+    :param world: The simulation world instance.
+    :param map_name: Map to load and apply settings to.
+    """
+
+    if map_name is None:
+    	print('Cannot load provided map')
+    	return
+
+    # Load the desired map
+    print(f"Loading map: {map_name}")
+    client.load_world(map_name)
+
+    # Get Settings
+    settings = world.get_settings()
+
+    # Set synchronous mode
+    settings.synchronous_mode = True
+    settings.fixed_delta_seconds = SIM_DT
+
+    # Set physics substepping
+    settings.substepping = False
+    # settings.max_substep_delta_time = 0.001  # max 1 ms per physics substep, swap to 0.01 if no neet of extreme physics realism
+    # settings.max_substeps = 10
+
+    world.apply_settings(settings)
+
+    # Disable TF publishing in CARLA to avoid conflicts.
+    world.set_publish_tf(
+        False)  # Autoware will be publishing TF information based on the URDF files of the vehicle and sensor kit.
+    # client.reload_world(False)  # reload map keeping the world settings
+
+
+def run_simulation_loop(world, target_time_scale=1.0, acceptable_lag=0.05, should_resync=False, ego=None,
+                        follow_ego=False):
```

Comment:
Suggestion: split a long argument list so that each argument is in a separate line.
This is consistent with the rest of the source code within that file.

```suggestion
def run_simulation_loop(
        world,
        target_time_scale=1.0,
        acceptable_lag=0.05,
        should_resync=False,
        ego=None,
        follow_ego=False):
```

Reply:
Resolved in #37